packages/webapp/pages/account/notifications.tsx

```diff
@@ -186,7 +186,7 @@ const AccountNotificationsPage = (): ReactElement => {
       logEvent({
         event_name: LogEvent.ScheduleStreakReminder,
         extra: JSON.stringify({
-          hour: readingTimeIndex,
+          hour: 20,
```

Comment:
why is this changed?

Reply:
It's supposed to be hardcoded 20 was a mistake from the previous implementation as you can't change it.